tracer/src/Datadog.Tracer.Native/Generated/generated_calltargets.g.h

```diff
@@ -0,0 +1,908 @@
+{(WCHAR*)WStr("StackExchange.Redis.StrongName"),(WCHAR*)WStr("StackExchange.Redis.RedisBatch"),(WCHAR*)WStr("ExecuteAsync"),g_callTargets_Sig_167,4,1,0,0,2,65535,65535,assemblyName,(WCHAR*)WStr("Datadog.Trace.ClrProfiler.AutoInstrumentation.Redis.StackExchange.RedisExecuteAsyncIntegration"),CallTargetKind::Default,1,165},
+{(WCHAR*)WStr("StackExchange.Redis.StrongName"),(WCHAR*)WStr("StackExchange.Redis.RedisBatch"),(WCHAR*)WStr("ExecuteAsync"),g_callTargets_Sig_166,5,2,0,0,2,65535,65535,assemblyName,(WCHAR*)WStr("Datadog.Trace.ClrProfiler.AutoInstrumentation.Redis.StackExchange.RedisExecuteAsyncIntegration_2_6_48"),CallTargetKind::Default,1,165},
+{(WCHAR*)WStr("StackExchange.Redis.StrongName"),(WCHAR*)WStr("StackExchange.Redis.RedisTransaction"),(WCHAR*)WStr("ExecuteAsync"),g_callTargets_Sig_167,4,1,0,0,2,65535,65535,assemblyName,(WCHAR*)WStr("Datadog.Trace.ClrProfiler.AutoInstrumentation.Redis.StackExchange.RedisExecuteAsyncIntegration"),CallTargetKind::Default,1,165},
+{(WCHAR*)WStr("StackExchange.Redis.StrongName"),(WCHAR*)WStr("StackExchange.Redis.RedisTransaction"),(WCHAR*)WStr("ExecuteAsync"),g_callTargets_Sig_166,5,2,0,0,2,65535,65535,assemblyName,(WCHAR*)WStr("Datadog.Trace.ClrProfiler.AutoInstrumentation.Redis.StackExchange.RedisExecuteAsyncIntegration_2_6_48"),CallTargetKind::Default,1,165},
+{(WCHAR*)WStr("Microsoft.AspNetCore.Diagnostics"),(WCHAR*)WStr("Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"),(WCHAR*)WStr("DisplayException"),g_callTargets_Sig_159,3,2,0,0,2,65535,65535,assemblyName,(WCHAR*)WStr("Datadog.Trace.ClrProfiler.AutoInstrumentation.StackTraceLeak.DeveloperExceptionPageMiddlewareIntegration"),CallTargetKind::Default,4,164},
+{(WCHAR*)WStr("Microsoft.AspNetCore.Diagnostics"),(WCHAR*)WStr("Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"),(WCHAR*)WStr("DisplayException"),g_callTargets_Sig_157,2,3,0,0,6,65535,65535,assemblyName,(WCHAR*)WStr("Datadog.Trace.ClrProfiler.AutoInstrumentation.StackTraceLeak.DeveloperExceptionPageMiddlewareIntegration_Pre_3_0_0"),CallTargetKind::Default,4,164},
+{(WCHAR*)WStr("Microsoft.AspNetCore.Diagnostics"),(WCHAR*)WStr("Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl"),(WCHAR*)WStr("DisplayException"),g_callTargets_Sig_157,2,7,0,0,8,65535,65535,assemblyName,(WCHAR*)WStr("Datadog.Trace.ClrProfiler.AutoInstrumentation.StackTraceLeak.DeveloperExceptionPageMiddlewareIntegration_Pre_3_0_0"),CallTargetKind::Default,4,164},
+{(WCHAR*)WStr("System.Web"),(WCHAR*)WStr("System.Web.HttpResponse"),(WCHAR*)WStr("WriteErrorMessage"),g_callTargets_Sig_267,3,4,0,0,4,65535,65535,assemblyName,(WCHAR*)WStr("Datadog.Trace.ClrProfiler.AutoInstrumentation.StackTraceLeak.HttpResponseIntegration"),CallTargetKind::Default,4,1},
```

Comment:
As for callsites, I wonder if we can omit the NETFX-only integrations in linux/macos, but that's an optimization for later if we want to consider it

Reply:
https://github.com/DataDog/dd-trace-dotnet/pull/6270